requirements.txt

```diff
@@ -2,7 +2,6 @@ boto
 django-bootstrap3
 django-debug-toolbar
 django-debug-toolbar-template-timings
-django-extensions
```

Comment:
@waseem-omar why are you getting rid of django-extensions? This is a useful package that provides shell_plus management command, among others. If its just because it wasn't updated yet, please check again.